CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 # CHANGELOG
 
 # Unreleased
-
+  * BREAKING: Plek will no longer try to fix malformed service names and will
+    instead raise an ArgumentError.
+  * Add convenience methods for `Plek.external_url_for`, `Plek.asset_root`
+    and `Plek.website_root` for consistency in methods.
+  * Breaking: Remove `Plek.find_uri`, `Plek#find_uri`, `Plek#asset_uri` and
```

Comment:
Super duper nit (😅) : consistent capitalisation of "breaking"

```suggestion
  * BREAKING: Remove `Plek.find_uri`, `Plek#find_uri`, `Plek#asset_uri` and
```

Reply:
Ah thanks! I'm glad you spotted as I'd have felt shame seeing that in changelogs all over Github